UKHomeOffice/sgmr-service
# SPCR-526 - Remove ZZZA/D as default UNLOCODES

### AC / Description of changes

ZZZA/D are not used by CBP and the way we have been using them has resulted in CBP not being able to match a location to it's ref data when we send them.

The solution will be to send port name when there is no UNLOCODE. The `sgmr-data-api` is being updated to check for `null` value in port field (`arrivalPort`, `departurePort`) and if null then send the `arrivalPortName`/`departurePortName` instead of the `arrivalPort`/`departurePort` (which contain the UNLOCODE.

This PR removes the use of ZZZA/D as default values and allows the arrival and departure port fields to be null.


### To Test

To test this specific PR
- enter a departure port location with a UNLOCODE (eg portsmouth)
- enter an arrival port location without one (eg Dover Marina)
- you should be able to continue through the form
- > on the Check your answers page you should see `Portsmouth (GB PME)` and `Dover Marina`
- go back to departure page, you should see `Portsmouth (GB PME)`
- go back to arrivals page, you should see `Dover Marina`
- submit voyage
- > it should submit without port error


### Notes

---

**Remember to merge to feature branches not `master`.**
